docs/reference/docs/update.asciidoc

```diff
@@ -197,8 +197,15 @@ The update operation supports similar parameters as the index API,
 including:
 
 [horizontal]
-`routing`::     Sets the routing that will be used to route the 
-                document to the relevant shard.
+`routing`::     Can't be used to update the routing of an existing document.
```

Comment:
I think this would read better by putting the warning about this not updating the routing/parent values as a second sentence here, and having the basic description of the options first.